test/models/concerns/contest_attributes_test.rb

```diff
@@ -78,16 +78,16 @@ class ContestAttributesTest < ActiveSupport::TestCase
       create(:short_problem, contest: c1, problem_no: i + 1)
     end
     7.times.each do |i|
-      create(:long_problem, contest: c1, problem_no: i + 1)
+      create(:long_problem, contest: c1, problem_no: i + 1, max_score: 10)
     end
     5.times.each do |i|
       create(:short_problem, contest: c2, problem_no: i + 1)
```

Comment:
Can you also put `correct_score: 4` here?

The idea is that even though it is already specified in `factories.rb` we want developers who read this test to understand this test easily without looking at any other files. Readability is always our main goal.